odoo/addons/base/migrations/11.0.1.3/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,399 @@
+---Fields in module 'base'---
+base         / ir.actions.act_url       / binding_model_id (many2one)   : NEW relation: ir.model
+base         / ir.actions.act_url       / binding_type (selection)      : NEW required: required, selection_keys: ['action', 'report'], req_default: function
+# NOTHING TO DO
+
+base         / ir.actions.act_url       / usage (char)                  : DEL 
+base         / ir.actions.act_window_close / usage (char)                  : DEL 
+base         / ir.actions.actions       / usage (char)                  : DEL 
+base         / ir.actions.client        / usage (char)                  : DEL 
+# NOTHING TO DO, obsolete column
+
+base         / ir.actions.act_window    / binding_model_id (many2one)   : NEW relation: ir.model
+base         / ir.actions.act_window    / binding_type (selection)      : NEW required: required, selection_keys: ['action', 'report'], req_default: function
```

Comment:
there's a lot to do for this: This fields (actually, this happens in the action's base model) replace the horrible ir.values model to link actions to models for the sidebar. You'll have to fill those values from the appropriate ir.values record